web/packages/teleterm/src/services/config/configService.ts

```diff
@@ -20,70 +20,65 @@ import { FileStorage } from 'teleterm/services/fileStorage';
 import { Platform } from 'teleterm/mainProcess/types';
 
 import { createConfigStore } from './configStore';
+import { getKeyboardShortcutSchema } from './getKeyboardShortcutSchema';
 
 const createAppConfigSchema = (platform: Platform) => {
   const defaultKeymap = getDefaultKeymap(platform);
   const defaultTerminalFont = getDefaultTerminalFont(platform);
 
-  // Important: all keys except 'usageReporting.enabled' are currently not
-  // configurable by the user. Before we let the user configure them,
-  // we need to set up some actual validation, so that for example
-  // arbitrary CSS cannot be injected into the app through font settings.
-  //
-  // However, we want them to be in the config schema, so we included
-  // them here, but we do not read their value from the stored config.
+  // `keymap.` prefix is used in `initUi.ts` in a predicate function.
   return z.object({
     'usageReporting.enabled': z.boolean().default(false),
-    'keymap.tab1': omitStoredConfigValue(
-      z.string().default(defaultKeymap['tab1'])
+    'keymap.tab1': getKeyboardShortcutSchema(platform).default(
```

Comment:
One of the features I don't think we've considered is supporting more than one shortcut per action. We can come back to this in the future, it's a rather niche use case but I guess it might be something that power users want (and the whole keyboard config feature is for them really).

Reply:
We can go back to it in the future (it should be easy to implement), with the number of possible actions we have currently, I don't feel it is needed enough.